addons/core/functions/fnc_getDefaultRadioClasses.sqf

```diff
@@ -19,48 +19,53 @@
         _classes = call TFAR_fnc_getDefaultRadioClasses;
 */
 
-private _defaultLRRadio = TFAR_DefaultRadio_Backpack_Independent;
-private _defaultPersonalRadio = TFAR_DefaultRadio_Personal_Independent;
-private _defaultRiflemanRadio = TFAR_DefaultRadio_Rifleman_Independent;
-private _defaultAirborneRadio = TFAR_DefaultRadio_Airborne_Independent;
+params ["_unit", TFAR_currentUnit];
 
-switch (TFAR_currentUnit call BIS_fnc_objectSide) do {
-    case west: {
-        _defaultLRRadio = TFAR_DefaultRadio_Backpack_West;
-        _defaultPersonalRadio = TFAR_DefaultRadio_Personal_West;
-        _defaultRiflemanRadio = TFAR_DefaultRadio_Rifleman_West;
-        _defaultAirborneRadio = TFAR_DefaultRadio_Airborne_West;
-    };
-    case east: {
-        _defaultLRRadio = TFAR_DefaultRadio_Backpack_East;
-        _defaultPersonalRadio = TFAR_DefaultRadio_Personal_East;
-        _defaultRiflemanRadio = TFAR_DefaultRadio_Rifleman_East;
-        _defaultAirborneRadio = TFAR_DefaultRadio_Airborne_East;
-    };
-};
-
-TFAR_tryResolveFactionClass =
-{
+private _fnc_tryResolveFactionClass = {
     params ["_prefix", "_default"];
-    private _faction = faction TFAR_currentUnit;
-    private _result = missionNamespace getVariable (_faction + "_" + _prefix + "_tf_faction_radio");
+    private _faction = faction _unit;
+    private _result = missionNamespace getVariable (format["%1_%2_tf_faction_radio", _faction, _prefix]);
 
-    if (!isNil "_result") exitWith {_result};
+    if (_result isEqualTo "") exitWith {_default};
+    if (!isNil "_result") exitWith {_result};    
 
     if (isText (configFile >> "CfgFactionClasses" >> _faction >> (_prefix + "_tf_faction_radio_api"))) exitWith {
```

Comment:
api stuff is deprecated. Mark with `//#Deprecated`

Reply:
done, this file should get a review for 1.1

Reply:
Everything #Deprecated will get review... Should:tm: